src/mono_font/mod.rs

```diff
@@ -113,39 +113,26 @@
 //!
 //! | Type | Screenshot |
 //! |------|------------|
-//! | [`Font6x8`] | ![6x8 font spritemap screenshot](https://raw.githubusercontent.com/embedded-graphics/embedded-graphics/master/data/font6x8.png) |
-//! | [`Font6x12`] | ![6x12 font spritemap screenshot](https://raw.githubusercontent.com/embedded-graphics/embedded-graphics/master/data/font6x12.png) |
-//! | [`Font8x16`] | ![8x16 font spritemap screenshot](https://raw.githubusercontent.com/embedded-graphics/embedded-graphics/master/data/font8x16.png) |
-//! | [`Font12x16`] | ![12x16 font spritemap screenshot](https://raw.githubusercontent.com/embedded-graphics/embedded-graphics/master/data/font12x16.png) |
-//! | [`Font24x32`] | The 24x32 font is a pixel doubled version of the 12x16 font. |
+//! | `Font6x8` | ![6x8 font spritemap screenshot](https://raw.githubusercontent.com/embedded-graphics/embedded-graphics/master/data/font6x8.png) |
```

Comment:
Could you add a `//! TODO: Replace with generated fonts table` or similar comment to this doc? That way it's visible when grepping for `TODO`, as well as sticking out when people view the docs if we forget to update them before release.

If you're planning on changing this very soon anyway, you can probably ignore this comment.

Reply:
I hope to get this done soon, but I've added the comment anyway and will also add it to the tracking issue.